src/annotator/test/frame-observer-test.js

```diff
@@ -140,10 +140,16 @@ describe('annotator/frame-observer', () => {
     it("doesn't trigger onFrameAdded when annotatable iframe is from a different domain", async () => {
       const iframe = createAnnotatableIFrame();
       iframe.setAttribute('src', 'http://cross-origin.dummy');
+      let expectedError;
 
-      await onDocumentReady(iframe);
+      try {
+        await onDocumentReady(iframe);
+      } catch (error) {
+        expectedError = error;
+      }
       await waitForFrameObserver();
```

Comment:
Unrelated to the changes in this PR, the `waitForFrameObserver` helper would benefit from a comment to clarify what exactly it is waiting for.

Reply:
Would this be OK?
`// FrameObserver waits a few milliseconds (DEBOUNCE_WAIT) to detect changes in the DOM`